### PR TITLE
Fixed wagtail dep

### PR DESCRIPTION
**Pull request by eviltnan**
https://github.com/torchbox/wagtail-markdown/issues/53

Tested on my project https://github.com/eviltnan/portfolio, everything seems to work fine.

I'd propose not to hard pin wagtail version, it's unprobable to break because of changes introduced in upcoming wagtail releases. Hard pin is preventing updating wagtail in a non-contradictory way.

### CODE REVIEW

**Comment by kevinhowbrook** [comment]
This assumes that the package will work on anything in the future, that doesn't seem right to me.
If anything we would want something like `Wagtail>=1.10,<2.5`.

On a more wider note, I think we should remove this dependency completely. Wagtail Markdown shouldn't be dictating what version of wagtail to install, seen as though it's only going to be installed once you have wagtail installed

**Comment by kevinhowbrook** [comment]
https://github.com/torchbox/wagtail-markdown/pull/56

**Comment by eviltnan** [comment]
@kevinhowbrook could you please release that fix on pipy?

File: setup.py [hunk 1]
```diff
@@ -32,7 +32,7 @@ def get_git_revision_hash():
 INSTALL_REQUIRES = [
     'Markdown>=2.6,<2.7',
     'bleach>=1.4.2,<2.2',
-    'Wagtail<2.3',
+    'Wagtail>=2.4',
```
Comment (kevinhowbrook) [inline setup.py]: I think we should just change this to `Wagtail>=1.10,<2.5` for now so we don't make any assumptions about future versions

**Comment by mfitzp** [comment]
This also works fine with 2.5 as far as I can tell.

**Comment by zoombear** [comment]
Is this ready to go?

**Comment by kevinhowbrook** [comment]
Fixed in https://github.com/torchbox/wagtail-markdown/pull/60
Thanks all

**Pull request closed without merging**